SingularityUI/app/components/requestForm/RequestForm.jsx

```diff
@@ -628,6 +630,15 @@ const RequestForm = (props) => {
     />
   );
 
+  const avoidShuffle = (
```

Comment:
One note on placement of this. We will likely want to make a button+modal for this in the top bar of the request detail page (example below) rather than putting it on the request form, for two reasons:
1) We hide the request form from developers in most cases, since, internally, the request json is managed by our deploy api
2) Given that the request form is meant to essentially represent the request json, and shuffle is under a separate root/api/object, it makes sense to separate shuffle a bit

side note, a nice to have, but maybe we could actually show the button on the task detail page as well when the task in question was killed due to shuffling? Would have to identify that based on the cleanup message in the status updates

Reply:
I've implemented the side note, but to avoid complicating the redux flow for the TaskDetail page the button will always prompt the user to disable shuffling for the request instead of being a toggle - no technical issues (safe to post the same request id to the blacklist multiple times), but maybe a slight source of confusion.